diff --git a/vendor/magento/framework/View/Element/AbstractBlock.php b/vendor/magento/framework/View/Element/AbstractBlock.php
index ea47c22..dfc42c9 100644
--- a/vendor/magento/framework/View/Element/AbstractBlock.php
+++ b/vendor/magento/framework/View/Element/AbstractBlock.php
@@ -667,6 +667,9 @@ abstract class AbstractBlock extends \Magento\Framework\DataObject implements Bl
      */
     public function toHtml()
     {
+        $string_to_log = $this->getNameInLayout() . ': Abstract toHtml - Block class: ' . get_class($this) . "\n";
+        $this->_logger->info($string_to_log);
+
         $this->_eventManager->dispatch('view_block_abstract_to_html_before', ['block' => $this]);
         if ($this->_scopeConfig->getValue(
             'advanced/modules_disable_output/' . $this->getModuleName(),
@@ -1042,6 +1045,9 @@ abstract class AbstractBlock extends \Magento\Framework\DataObject implements Bl
      */
     public function getCacheKey()
     {
+        $string_to_log = $this->getNameInLayout() . ' getCacheKey - Block: ' . get_class($this) . "\n";
+        $this->_logger->info($string_to_log);
+
         if ($this->hasData('cache_key')) {
             return static::CACHE_KEY_PREFIX . $this->getData('cache_key');
         }
diff --git a/vendor/magento/framework/View/Element/BlockFactory.php b/vendor/magento/framework/View/Element/BlockFactory.php
index f159a9f..ecafffa 100644
--- a/vendor/magento/framework/View/Element/BlockFactory.php
+++ b/vendor/magento/framework/View/Element/BlockFactory.php
@@ -22,6 +22,8 @@ class BlockFactory
      */
     protected $objectManager;
 
+    protected $_logger = null;
+
     /**
      * Constructor
      *
@@ -43,6 +45,9 @@ class BlockFactory
     public function createBlock($blockName, array $arguments = [])
     {
         $blockName = ltrim($blockName, '\\');
+
+        $this->_getLogger()->info("Creating block: " . $blockName);
+
         $block = $this->objectManager->create($blockName, $arguments);
         if (!$block instanceof BlockInterface) {
             throw new \LogicException($blockName . ' does not implement BlockInterface');
@@ -52,4 +57,14 @@ class BlockFactory
         }
         return $block;
     }
+
+    protected function _getLogger()
+    {
+        if (is_null($this->_logger))
+        {
+            $this->_logger = $this->objectManager->get('\Psr\Log\LoggerInterface');
+        }
+
+        return $this->_logger;
+    }
 }
diff --git a/vendor/magento/framework/View/Element/Template.php b/vendor/magento/framework/View/Element/Template.php
index e9f164c..b3359b5 100644
--- a/vendor/magento/framework/View/Element/Template.php
+++ b/vendor/magento/framework/View/Element/Template.php
@@ -261,6 +261,9 @@ class Template extends AbstractBlock
      */
     public function fetchView($fileName)
     {
+        $string_to_log = $this->getNameInLayout() . ': fetchView Template: ' . $this->getTemplateFile() . "\n";
+        $this->_logger->info($string_to_log);
+
         $relativeFilePath = $this->getRootDirectory()->getRelativePath($fileName);
         \Magento\Framework\Profiler::start(
             'TEMPLATE:' . $fileName,
diff --git a/vendor/magento/framework/View/Layout.php b/vendor/magento/framework/View/Layout.php
index eeba748..b8847e6 100644
--- a/vendor/magento/framework/View/Layout.php
+++ b/vendor/magento/framework/View/Layout.php
@@ -588,7 +588,21 @@ class Layout extends \Magento\Framework\Simplexml\Config implements \Magento\Fra
     protected function _renderUiComponent($name)
     {
         $uiComponent = $this->getUiComponent($name);
-        return $uiComponent ? $uiComponent->toHtml() : '';
+
+        if ($uiComponent)
+        {
+            $string_to_log = $name . ' UiComponent before toHtml' . "\n";
+            $this->logger->info($string_to_log);
+
+            $html_to_return = $uiComponent->toHtml();
+
+            $string_to_log = $name . ' UiComponent after toHtml' . "\n";
+            $this->logger->info($string_to_log);
+
+            return $html_to_return;
+        }
+
+        return '';
     }
 
     /**
@@ -602,9 +616,17 @@ class Layout extends \Magento\Framework\Simplexml\Config implements \Magento\Fra
     {
         $html = '';
         $children = $this->getChildNames($name);
+
+        $string_to_log = $name . ' Container before render' . "\n";
+        $this->logger->info($string_to_log);
+
         foreach ($children as $child) {
             $html .= $this->renderElement($child, $useCache);
         }
+
+        $string_to_log = $name . ' Container after render' . "\n";
+        $this->logger->info($string_to_log);
+
         if ($html == '' || !$this->structure->getAttribute($name, Element::CONTAINER_OPT_HTML_TAG)) {
             return $html;
         }
diff --git a/vendor/magento/framework/View/Layout/Generator/Block.php b/vendor/magento/framework/View/Layout/Generator/Block.php
index 27af40f..95d7f4a 100644
--- a/vendor/magento/framework/View/Layout/Generator/Block.php
+++ b/vendor/magento/framework/View/Layout/Generator/Block.php
@@ -252,6 +252,10 @@ class Block implements Layout\GeneratorInterface
         $block = $this->getBlockInstance($block, $arguments);
         $block->setType(get_class($block));
         $block->setNameInLayout($name);
+
+        $string_to_log = $name . " was created - class: " . get_class($block);
+        $this->logger->info($string_to_log);
+
         $block->addData(isset($arguments['data']) ? $arguments['data'] : []);
         return $block;
     }
diff --git a/vendor/magento/framework/View/Model/Layout/Merge.php b/vendor/magento/framework/View/Model/Layout/Merge.php
index 239d416..011a8c8 100644
--- a/vendor/magento/framework/View/Model/Layout/Merge.php
+++ b/vendor/magento/framework/View/Model/Layout/Merge.php
@@ -274,9 +274,15 @@ class Merge implements \Magento\Framework\View\Layout\ProcessorInterface
     {
         if (is_array($handleName)) {
             foreach ($handleName as $name) {
+
+                $this->logger->info('Added handle: ' . $name);
+
                 $this->handles[$name] = 1;
             }
         } else {
+
+            $this->logger->info('Added handle: ' . $handleName);
+
             $this->handles[$handleName] = 1;
         }
         return $this;
