diff --git a/vendor/magento/framework/View/Element/AbstractBlock.php b/vendor/magento/framework/View/Element/AbstractBlock.php
index 64897b4..4ad04e1 100644
--- a/vendor/magento/framework/View/Element/AbstractBlock.php
+++ b/vendor/magento/framework/View/Element/AbstractBlock.php
@@ -639,6 +639,12 @@ abstract class AbstractBlock extends \Magento\Framework\DataObject implements Bl
      */
     public function toHtml()
     {
+        $this->_logger->info('Abstract toHtml');
+        $this->_logger->info('Block: ' . get_class($this));
+        $this->_logger->info('Layout Name' . $this->getNameInLayout());
+
+
+
         $this->_eventManager->dispatch('view_block_abstract_to_html_before', ['block' => $this]);
         if ($this->_scopeConfig->getValue(
             'advanced/modules_disable_output/' . $this->getModuleName(),
@@ -962,6 +968,11 @@ abstract class AbstractBlock extends \Magento\Framework\DataObject implements Bl
      */
     public function getCacheKey()
     {
+        $this->_logger->info('getCacheKey');
+        $this->_logger->info('Block: ' . get_class($this));
+        $this->_logger->info('Layout Name' . $this->getNameInLayout());
+
+
         if ($this->hasData('cache_key')) {
             return static::CACHE_KEY_PREFIX . $this->getData('cache_key');
         }
diff --git a/vendor/magento/framework/View/Element/BlockFactory.php b/vendor/magento/framework/View/Element/BlockFactory.php
index 3244f41..f3b8adc 100644
--- a/vendor/magento/framework/View/Element/BlockFactory.php
+++ b/vendor/magento/framework/View/Element/BlockFactory.php
@@ -19,14 +19,17 @@ class BlockFactory
      */
     protected $objectManager;
 
+    protected $_logger;
+
     /**
      * Constructor
      *
      * @param ObjectManagerInterface $objectManager
      */
-    public function __construct(ObjectManagerInterface $objectManager)
+    public function __construct(ObjectManagerInterface $objectManager, \Psr\Log\LoggerInterface $logger)
     {
         $this->objectManager = $objectManager;
+        $this->_logger = $logger;
     }
 
     /**
@@ -40,6 +43,9 @@ class BlockFactory
     public function createBlock($blockName, array $arguments = [])
     {
         $blockName = ltrim($blockName, '\\');
+
+        $this->_logger->info($blockName);
+
         $block = $this->objectManager->create($blockName, $arguments);
         if (!$block instanceof BlockInterface) {
             throw new \LogicException($blockName . ' does not implement BlockInterface');
diff --git a/vendor/magento/framework/View/Element/Template.php b/vendor/magento/framework/View/Element/Template.php
index f011c1e..770545d 100644
--- a/vendor/magento/framework/View/Element/Template.php
+++ b/vendor/magento/framework/View/Element/Template.php
@@ -243,6 +243,8 @@ class Template extends AbstractBlock
      */
     public function fetchView($fileName)
     {
+        $this->_logger->info('fetchView Template: ' . $this->getTemplateFile());
+
         $relativeFilePath = $this->getRootDirectory()->getRelativePath($fileName);
         \Magento\Framework\Profiler::start(
             'TEMPLATE:' . $fileName,
@@ -276,6 +278,7 @@ class Template extends AbstractBlock
         if (!$this->getTemplate()) {
             return '';
         }
+
         return $this->fetchView($this->getTemplateFile());
     }
 
diff --git a/vendor/magento/framework/View/Model/Layout/Merge.php b/vendor/magento/framework/View/Model/Layout/Merge.php
index d419f44..4f2f0e2 100644
--- a/vendor/magento/framework/View/Model/Layout/Merge.php
+++ b/vendor/magento/framework/View/Model/Layout/Merge.php
@@ -235,9 +235,15 @@ class Merge implements \Magento\Framework\View\Layout\ProcessorInterface
     {
         if (is_array($handleName)) {
             foreach ($handleName as $name) {
+
+                $this->logger->info('Added handle: ' . $name);
+
                 $this->handles[$name] = 1;
             }
         } else {
+
+            $this->logger->info('Added handle: ' . $handleName);
+
             $this->handles[$handleName] = 1;
         }
         return $this;
