diff --git a/vendor/magento/framework/View/Element/Template.php b/vendor/magento/framework/View/Element/Template.php
index d756dc1..136ab32 100644
--- a/vendor/magento/framework/View/Element/Template.php
+++ b/vendor/magento/framework/View/Element/Template.php
@@ -276,11 +276,6 @@ class Template extends AbstractBlock
         if (!$this->getTemplate()) {
             return '';
         }
-
-        $this->_logger->info(get_class($this));
-        $this->_logger->info($this->getNameInLayout());
-        $this->_logger->info($this->getTemplateFile());
-
         return $this->fetchView($this->getTemplateFile());
     }
 
