diff --git a/vendor/magento/framework/View/Element/AbstractBlock.php b/vendor/magento/framework/View/Element/AbstractBlock.php
index dfc42c9..ea47c22 100644
--- a/vendor/magento/framework/View/Element/AbstractBlock.php
+++ b/vendor/magento/framework/View/Element/AbstractBlock.php
@@ -667,9 +667,6 @@ abstract class AbstractBlock extends \Magento\Framework\DataObject implements Bl
      */
     public function toHtml()
     {
-        $string_to_log = $this->getNameInLayout() . ': Abstract toHtml - Block class: ' . get_class($this) . "\n";
-        $this->_logger->info($string_to_log);
-
         $this->_eventManager->dispatch('view_block_abstract_to_html_before', ['block' => $this]);
         if ($this->_scopeConfig->getValue(
             'advanced/modules_disable_output/' . $this->getModuleName(),
@@ -1045,9 +1042,6 @@ abstract class AbstractBlock extends \Magento\Framework\DataObject implements Bl
      */
     public function getCacheKey()
     {
-        $string_to_log = $this->getNameInLayout() . ' getCacheKey - Block: ' . get_class($this) . "\n";
-        $this->_logger->info($string_to_log);
-
         if ($this->hasData('cache_key')) {
             return static::CACHE_KEY_PREFIX . $this->getData('cache_key');
         }
diff --git a/vendor/magento/framework/View/Element/BlockFactory.php b/vendor/magento/framework/View/Element/BlockFactory.php
index ecafffa..f159a9f 100644
--- a/vendor/magento/framework/View/Element/BlockFactory.php
+++ b/vendor/magento/framework/View/Element/BlockFactory.php
@@ -22,8 +22,6 @@ class BlockFactory
      */
     protected $objectManager;
 
-    protected $_logger = null;
-
     /**
      * Constructor
      *
@@ -45,9 +43,6 @@ class BlockFactory
     public function createBlock($blockName, array $arguments = [])
     {
         $blockName = ltrim($blockName, '\\');
-
-        $this->_getLogger()->info("Creating block: " . $blockName);
-
         $block = $this->objectManager->create($blockName, $arguments);
         if (!$block instanceof BlockInterface) {
             throw new \LogicException($blockName . ' does not implement BlockInterface');
@@ -57,14 +52,4 @@ class BlockFactory
         }
         return $block;
     }
-
-    protected function _getLogger()
-    {
-        if (is_null($this->_logger))
-        {
-            $this->_logger = $this->objectManager->get('\Psr\Log\LoggerInterface');
-        }
-
-        return $this->_logger;
-    }
 }
diff --git a/vendor/magento/framework/View/Element/Template.php b/vendor/magento/framework/View/Element/Template.php
index b3359b5..e9f164c 100644
--- a/vendor/magento/framework/View/Element/Template.php
+++ b/vendor/magento/framework/View/Element/Template.php
@@ -261,9 +261,6 @@ class Template extends AbstractBlock
      */
     public function fetchView($fileName)
     {
-        $string_to_log = $this->getNameInLayout() . ': fetchView Template: ' . $this->getTemplateFile() . "\n";
-        $this->_logger->info($string_to_log);
-
         $relativeFilePath = $this->getRootDirectory()->getRelativePath($fileName);
         \Magento\Framework\Profiler::start(
             'TEMPLATE:' . $fileName,
diff --git a/vendor/magento/framework/View/Layout.php b/vendor/magento/framework/View/Layout.php
index b8847e6..eeba748 100644
--- a/vendor/magento/framework/View/Layout.php
+++ b/vendor/magento/framework/View/Layout.php
@@ -588,21 +588,7 @@ class Layout extends \Magento\Framework\Simplexml\Config implements \Magento\Fra
     protected function _renderUiComponent($name)
     {
         $uiComponent = $this->getUiComponent($name);
-
-        if ($uiComponent)
-        {
-            $string_to_log = $name . ' UiComponent before toHtml' . "\n";
-            $this->logger->info($string_to_log);
-
-            $html_to_return = $uiComponent->toHtml();
-
-            $string_to_log = $name . ' UiComponent after toHtml' . "\n";
-            $this->logger->info($string_to_log);
-
-            return $html_to_return;
-        }
-
-        return '';
+        return $uiComponent ? $uiComponent->toHtml() : '';
     }
 
     /**
@@ -616,17 +602,9 @@ class Layout extends \Magento\Framework\Simplexml\Config implements \Magento\Fra
     {
         $html = '';
         $children = $this->getChildNames($name);
-
-        $string_to_log = $name . ' Container before render' . "\n";
-        $this->logger->info($string_to_log);
-
         foreach ($children as $child) {
             $html .= $this->renderElement($child, $useCache);
         }
-
-        $string_to_log = $name . ' Container after render' . "\n";
-        $this->logger->info($string_to_log);
-
         if ($html == '' || !$this->structure->getAttribute($name, Element::CONTAINER_OPT_HTML_TAG)) {
             return $html;
         }
diff --git a/vendor/magento/framework/View/Layout/Generator/Block.php b/vendor/magento/framework/View/Layout/Generator/Block.php
index 95d7f4a..27af40f 100644
--- a/vendor/magento/framework/View/Layout/Generator/Block.php
+++ b/vendor/magento/framework/View/Layout/Generator/Block.php
@@ -252,10 +252,6 @@ class Block implements Layout\GeneratorInterface
         $block = $this->getBlockInstance($block, $arguments);
         $block->setType(get_class($block));
         $block->setNameInLayout($name);
-
-        $string_to_log = $name . " was created - class: " . get_class($block);
-        $this->logger->info($string_to_log);
-
         $block->addData(isset($arguments['data']) ? $arguments['data'] : []);
         return $block;
     }
diff --git a/vendor/magento/framework/View/Model/Layout/Merge.php b/vendor/magento/framework/View/Model/Layout/Merge.php
index 011a8c8..239d416 100644
--- a/vendor/magento/framework/View/Model/Layout/Merge.php
+++ b/vendor/magento/framework/View/Model/Layout/Merge.php
@@ -274,15 +274,9 @@ class Merge implements \Magento\Framework\View\Layout\ProcessorInterface
     {
         if (is_array($handleName)) {
             foreach ($handleName as $name) {
-
-                $this->logger->info('Added handle: ' . $name);
-
                 $this->handles[$name] = 1;
             }
         } else {
-
-            $this->logger->info('Added handle: ' . $handleName);
-
             $this->handles[$handleName] = 1;
         }
         return $this;
