diff --git a/app/Mage.php b/app/Mage.php
index 2834e6a..2a93c04 100644
--- a/app/Mage.php
+++ b/app/Mage.php
@@ -786,7 +786,7 @@ final class Mage
     {
         $message = sprintf("[%s]%s", gethostname(), $message);
         if (!self::getConfig() || empty($file) || strtolower($file) == 'system.log') {
-            //return;
+            return;
         }
 
         try {
diff --git a/app/code/core/Mage/Core/Block/Template.php b/app/code/core/Mage/Core/Block/Template.php
index 9c7f20d..1eb2004 100644
--- a/app/code/core/Mage/Core/Block/Template.php
+++ b/app/code/core/Mage/Core/Block/Template.php
@@ -280,10 +280,6 @@ HTML;
      */
     protected function _toHtml()
     {
-        Mage::log(get_class($this));
-        Mage::log($this->getNameInLayout());
-        Mage::log($this->getTemplateFile());
-
         if (!$this->getTemplate()) {
             return '';
         }
diff --git a/app/code/core/Mage/Core/Model/Layout/Update.php b/app/code/core/Mage/Core/Model/Layout/Update.php
index 4c52eaa..642b810 100644
--- a/app/code/core/Mage/Core/Model/Layout/Update.php
+++ b/app/code/core/Mage/Core/Model/Layout/Update.php
@@ -126,8 +126,6 @@ class Mage_Core_Model_Layout_Update
 
     public function addHandle($handle)
     {
-        Mage::log($handle);
-
         if (is_array($handle)) {
             foreach ($handle as $h) {
                 $this->_handles[$h] = 1;
